__tests__/ChangePasswordScreen.test.js

```diff
@@ -0,0 +1,114 @@
+import "react-native";
+import React from "react";
+import { mount, shallow } from "enzyme";
+import { act, fireEvent, render, waitFor } from "@testing-library/react-native";
+import ChangePasswordScreen from "../screens/ChangePasswordScreen";
+
+// Note: this is just for use with Jest snapshot testing
+// and comes packaged with react-native init project.
+// You do not need this if using Enzyme 'toMatchSnapshot' etc.
+
+import renderer from "react-test-renderer";
+
+describe("Testing ChangePasswordScreen", () => {
+  it("renders correctly, test using Jest", () => {
+    renderer.create(<ChangePasswordScreen />);
+  });
+
+  it("matches the snapshot", () => {
+    const tree = renderer.create(<ChangePasswordScreen />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it("text on change password screen", () => {
+    const wrapper = shallow(<ChangePasswordScreen />);
+    const text = wrapper.find("Text").render().text();
+    expect(text).toEqual("Change your password");
+  });
+
+  it("should say the correct value", () => {
+    const wrapper = mount(<ChangePasswordScreen />);
+
+    const currentPassword = wrapper.find("TextInput").find("#currentPassword");
+    const newPassword = wrapper.find("TextInput").find("#newPassword");
+    const confirmPassword = wrapper.find("TextInput").find("#confirmPassword");
+
+    currentPassword.props().value = "Password";
+    newPassword.props().value = "Peyton12";
+    confirmPassword.props().value = "Peyton12";
+
+    expect(currentPassword.props().value).toEqual("Password");
+    expect(newPassword.props().value).toEqual("Peyton12");
+    expect(confirmPassword.props().value).toEqual("Peyton12");
+  });
+
+  it('should say "Please make sure new passwords match!" when the new passwords dont match', () => {
+    const wrapper = mount(<ChangePasswordScreen />);
+
+    const errorMessage = wrapper.find(".errorMessage");
+
+    errorMessage.props().error = "Please make sure new passwords match!";
+    errorMessage.props().visible = true;
+
+    // button.simulate("click");
+
+    expect(errorMessage.props().error).toEqual(
+      "Please make sure new passwords match!"
+    );
+  });
+
+  // it('should say "Please make sure new passwords match!" when the new passwords dont match', () => {
```

Comment:
If this commented-out code is going to be permanently discarded, let's remote it rather than leave it in commented out.

Reply:
Yes those were test using enzyme which I couldn't finish. I will trash them!